app/components/post_draft/post_input/post_input.js

```diff
@@ -302,6 +302,7 @@ export default class PostInput extends PureComponent {
                 disableFullscreenUI={true}
                 editable={!readonly}
                 onPaste={onPasteFiles}
+                maxHeight={100}
```

Comment:
This is already applied a few lines above with a different value. Also can you explain why you choose 100pt?

Reply:
Sorry I missed that..i will change the value in the previous code. I choose 100 because I found this value appropriate since it was working fine on android device, iphone 8 as well as iphone 11